packages/frontend/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import { configureStore } from "./state/store";
 
 const { store } = configureStore();
 
-const container = document.getElementById("root");
+const container = document.getElementById("root")!;
```

Comment:
nani

Reply:
yea sorry i added this – typescript was complaining

Reply:
its cuz of stricter types in react 18